io.catenax.mandatory_dismantling/1.0.0/MandatoryDismantling.ttl

```diff
@@ -0,0 +1,994 @@
+   samm:properties ( :isPresent ) .
+
+:AirConditioningFluidEntity a samm:Entity ;
+   samm:preferredName "Air Conditioning Fluid Entity"@en ;
+   samm:description "Encapsulates the Air Conditioning Fluid details"@en ;
+   samm:properties ( :itemType ) .
+
+:OtherFluidEntity a samm:Entity ;
+   samm:preferredName "Other Fluid Entity"@en ;
+   samm:description "Encapsulates the other Fluid details"@en ;
+   samm:properties ( :itemType ) .
+
+:AirbagEntity a samm:Entity ;
+   samm:preferredName "Airbag Entity"@en ;
+   samm:description "Encapsulates the Airbag details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:LiquefiedPetroleumGasTankEntity a samm:Entity ;
+   samm:preferredName "Liquefied Petroleum Gas Tank Entity"@en ;
+   samm:description "Encapsulates the Liquefied Petroleum Gas Tank details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:CompressedNaturalGasTankEntity a samm:Entity ;
+   samm:preferredName "Compressed Natural Gas Tank Entity"@en ;
+   samm:description "Encapsulates the Compressed Natural Gas Tank details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:HydrogenTankEntity a samm:Entity ;
+   samm:preferredName "Hydrogen Tank Entity"@en ;
+   samm:description "Encapsulates the Hydrogen Tank details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:ExplosivePartEntity a samm:Entity ;
+   samm:preferredName "Explosive Part Entity"@en ;
+   samm:description "Encapsulates the Explosive Part details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:AirConditioningSystemEntity a samm:Entity ;
+   samm:preferredName "Air Conditioning System Entity"@en ;
+   samm:description "Encapsulates the Air Conditioning System details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:RefrigerantEntity a samm:Entity ;
+   samm:preferredName "Refrigerant Entity"@en ;
+   samm:description "Encapsulates the Refrigerant details"@en ;
+   samm:properties ( :itemDetail :isPresent ) .
+
+:ComponentContainingMercuryEntity a samm:Entity ;
+   samm:preferredName "Component Containing Mercury Entity"@en ;
+   samm:description "Encapsulates the Component Containing Mercury details"@en ;
+   samm:properties ( :isPresent :itemDetail ) .
+
+:PartDetailEntity a samm:Entity ;
+   samm:preferredName "Part Detail Entity"@en ;
+   samm:description "Encapsulates the details of parent part"@en ;
+   samm:properties ( :number :location :removalInstruction ) .
+
+:BatteryDetailEntity a samm:Entity ;
+   samm:preferredName "Battery Detail Entity"@en ;
+   samm:description "Encapsulates the Battery detail."@en ;
+   samm:properties ( :batteryType :dischargeInstruction :requiredTool :weight ) .
+
+:MotorDetailEntity a samm:Entity ;
+   samm:preferredName "Motor Detail Entity"@en ;
+   samm:description "Encapsulates the Motor detail."@en ;
+   samm:properties ( :weight :requiredTool :permanentMagnet ) .
+
+:itemType a samm:Property ;
+   samm:preferredName "itemType"@en ;
```

Comment:
preferredName should be human readable and follow normal orthography (e.g., no camel case but normal word separation).